AI engineering category (#12665)
PostHog/posthog.com: contents/docs/ai-engineering/wizard.mdx

---
title: Wizard 
---

import WizardInstall from '../getting-started/_snippets/wizard.mdx'

> The PostHog wizard helps you instantly add PostHog to your application codebase using AI. It's an agentic CLI tool that handles the entire PostHog installation and integration process on your behalf.

Reviewer: Unfortunately, not instant
```suggestion
The PostHog AI installation wizard helps you easily add PostHog to your application codebase using AI. It's an agentic CLI tool (AKA [envoy](/blog/envoy-wizard-llm-agent) that handles the entire PostHog installation and integration process on your behalf.
```

```bash

> npx @posthog/wizard

Reviewer: Is there an us/eu thing we show account for here? If it works fine without, all good.

> npx @posthog/wizard

Reviewer: also the `-y` param?
```suggestion
npx @posthog/wizard@latest
```

```

With a single command, the wizard automatically: 

1. Authenticates your PostHog account

> 2. Installs PostHog SDK packages

Reviewer: ```suggestion
2. Installs the relevant PostHog SDK(s)
```
